bc_obps/service/tests/test_operation_service_v2.py

```diff
@@ -1443,17 +1441,17 @@ def test_gets_unfiltered_sorted_list_for_industry_user():
         baker.make_recipe(
             'registration.tests.utils.operation_designated_operator_timeline',
             operator=approved_user_operator.operator,
-            status=OperationDesignatedOperatorTimeline.Statuses.TEMPORARILY_SHUTDOWN,
+            end_date=None,
         )
         baker.make_recipe(
             'registration.tests.utils.operation_designated_operator_timeline',
             operator=approved_user_operator.operator,
-            status=OperationDesignatedOperatorTimeline.Statuses.TRANSFERRED,
+            end_date=None,
         )
         baker.make_recipe(
             'registration.tests.utils.operation_designated_operator_timeline',
             operator=approved_user_operator.operator,
-            status=OperationDesignatedOperatorTimeline.Statuses.ACTIVE,
+            end_date=None,
         )
```

Comment:
Interesting that these three don't trigger the unique constraint. Does it only happen if you use _quantity?

Reply:
yes, it was only happening when using `_quantity`. When I tried 

```
baker.make_recipe('operation_designated_operator_timeline', operation=baker.make_recipe('operation'), end_date=None, _quantity=20)
```

it was only creating 1 mock operation, and then trying to assign 20 active timelines for that 1 operation, which isn't allowed because we have a Django constraint that forbids it.

Reply:
that's why I instead had to first make 20 different operations, and then loop through them to create one timeline for each unique operation